src/vector/_methods.py

```diff
@@ -3209,49 +3214,55 @@ def to_Vector4D(
         self,
         *,
         z: float | None = None,
+        pz: float | None = None,
         theta: float | None = None,
         eta: float | None = None,
         t: float | None = None,
+        e: float | None = None,
+        E: float | None = None,
+        energy: float | None = None,
         tau: float | None = None,
+        m: float | None = None,
+        M: float | None = None,
+        mass: float | None = None,
```

Comment:
If somebody passes both `z` and `pz`, or both `z` and `theta` for that matter, does the function complain? It shouldn't just pick one and use it—providing (possibly) conflicting arguments should be an error, to prevent silent mistakes.

Reply:
Thanks for the review! Yes, the functions was checking for multiple generic coordinates, and I've added momentum coordinated to that check too.